plugins/com.google.cloud.tools.eclipse.util/build.properties

```diff
@@ -3,6 +3,7 @@ output.. = target/classes/
 bin.includes = META-INF/,\
                .,\
                plugin.xml,\
```

Comment:
I think you're missing the template directory here too, which might account for why the templates aren't found.